tools/compiletest/src/common.rs

```diff
@@ -145,6 +146,13 @@ pub struct Config {
     /// updating multiple tests. Users should still manually edit the files after to only keep
     /// relevant expectations.
     pub fix_expected: bool,
+
+    /// Whether we should measure and limit the time of a test.
+    pub time_opts: Option<TestTimeOptions>,
+
+    /// Extra arguments to be passed to Kani in this regression.
+    /// Note that there is no validation done whether these flags conflict with existing flags.
+    pub extra_args: Vec<String>,
```

Comment:
Could you add an example of a kani argument that we could add with this `extra_args` flag and maybe a simple invocation example?